pkg/validation/app.go

```diff
@@ -33,6 +37,28 @@ const (
 	nameMaxLength = 53
 )
 
+var (
+	fixedProtectedNamespaces = map[string]bool{
+		"flux-giantswarm": true,
+		"giantswarm":      true,
+		"kube-system":     true,
+		"monitoring":      true,
```

Comment:
I know it's **super** legacy, but let's add `draughtsman` here. It contains some secrets.